downstream/modules/platform/proc-choosing-obtaining-installer.adoc

```diff
@@ -15,7 +15,9 @@ A valid Red Hat customer account is required to access {PlatformName} installer
 
 .Installing with internet access
 
-Choose the {PlatformName} installer if your Red Hat Enterprise Linux environment is connected to the internet. Installing with internet access will retrieve the latest required repositories, packages, and dependencies.
+Choose the {PlatformName} (AAP) installer if your Red Hat Enterprise Linux environment is connected to the internet. Installing with internet access will retrieve the latest required repositories, packages, and dependencies. Choose one of the following ways to set up your AAP installer.
```

Comment:
Suggest active voice for second sentence: Installing with internet access retrieves...

Reply:
Thanks, @dcdacosta. Can you please provide a final review and then squash and merge?